Makefile

```diff
@@ -21,4 +21,4 @@ generate-spec:
 	go run cmd/spec/main.go
 
 validate-topics:
```

Comment:
We probably want to rename the script as well is we run multiple validate scripts